realtime/utils.js

```diff
@@ -211,7 +223,7 @@ function shouldIEmitThisObj(nspString, obj) {
      * subjectAbsolutePath in it, so we do not have to check for the filter
      * conditions and we just need to return true.
      */
-    if (nspComponents.length < 2) {
+    if (nspComponents.length <= 2) {
```

Comment:
Why this change?

Reply:
bot namespace only has 2 components

Reply:
This is contained within a check for subject absolutePath -- does your use case even get inside of this "if"?